features/netsocket/DNS.h

```diff
@@ -99,6 +99,25 @@ class DNS {
      *  @return         NSAPI_ERROR_OK on success, negative error code on failure.
      */
     virtual nsapi_error_t add_dns_server(const SocketAddress &address, const char *interface_name = NULL) = 0;
+
```

Comment:
Now you have two interface functions 
* one getting predefined (for symmetry stake)
* One getting the stack defined

But none for getting the actual order that the server uses when going through the list of servers

@kjbracey-arm could you give a input on how this should be done as I am not sure what is the interfaces we should have. Our only wish is to get all servers that are available, but now we are adding multiple APIs and there should be some consistency

Reply:
All we need is to get list of servers that DNS Client will use that includes the defaults the stacks and anything else possible.